src/styles/components/page-intro.scss

```diff
@@ -0,0 +1,26 @@
+@import "../settings";
+@import "../colours";
+
+.page-intro {
+  .intro-content {
+    -webkit-column-count: 2; /* Chrome, Safari, Opera */
+    -moz-column-count: 2; /* Firefox */
+    column-count: 2;
+  }
+  .intro-links {
+    text-align: right;
+    a {
+      margin-right: $global-margin;
+    }
+  }
+  button {
+    cursor: pointer;
+    color: $colour-black;
+  }
+  button:focus {
```

Comment:
The `button:focuse` should be moved inside the `button` block -- above, and should be written as:

    button {
      &:focus {
        ...
      }
    }

Reply:
Woops